lib/util/fs/index.ts

```diff
@@ -105,6 +106,27 @@ export function localPathExists(pathName: string): Promise<boolean> {
     .catch(() => false);
 }
 
+/**
+ * Tries to find one of the provided `fileNames` next to `existingFileNameWithPath`,
+ * then in its parent directory, then in the grandparent directory,
+ * until we reach the top-level of the `localDir` from the config.
+ */
+export async function findUpInsidePath(
```

Comment:
I think `findLocalSiblingOrParent` will already do what you need

Reply:
removed in 28a36aa130334e844a89884a6cb31c3fbf528659